virttest/remote.py

```diff
@@ -286,7 +286,7 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
         if identity_file:
             cmd += (" -i %s" % identity_file)
         else:
-            cmd += " -o PreferredAuthentications=password"
+            cmd += " -o PreferredAuthentications=password,keyboard-interactive"
```

Comment:
With adding keyboard-interactive, it may bring a additional try if ssh login fail under PreferredAuthentications=password.
This may impact some cases with intended failure login.

Reply:
@luckyh @ldoktor Could you help to check if this change has any side-effect for other people?  Thanks!

Reply:
The reason of the change is that VMX server doesn't support password authentication by default. And i didn't see any side-effect from my point.

Reply:
Hello guys, I looked at it and keyboard-interactive is not usually enabled by default on RHEL/Fedora systems. I'm not sure how the `handle_prompts` will react in case it is supported but at this point it looks safe-ish (I can't really say it won't break anything, but I don't think it'd directly affect me).

Reply:
@ldoktor Thanks for you reply.
@chunfuwen Can we merge it and see if anyone complains about it?

Reply:
I added a new optional parameter which can pass real preferredauthentication value to the function. By default, it still use 'password'.